File: articles/machine-learning-as-algorithmic-inference/java/ThresholdReview.java
public class ThresholdReview {
    public static void main(String[] args) {
        double tp = 80.0, fp = 25.0, tn = 140.0, fn = 35.0;
        double total = tp + fp + tn + fn;
        System.out.printf("accuracy=%.6f%n", (tp + tn) / total);
        System.out.printf("precision=%.6f%n", tp / (tp + fp));
        System.out.printf("recall=%.6f%n", tp / (tp + fn));
    }
}

File: articles/machine-learning-as-algorithmic-inference/fortran/threshold_review.f90
program threshold_review
  implicit none
  real :: tp, fp, tn, fn, total
  tp = 80.0
  fp = 25.0
  tn = 140.0
  fn = 35.0
  total = tp + fp + tn + fn
  print '(A,F8.6)', 'accuracy=', (tp + tn) / total
  print '(A,F8.6)', 'precision=', tp / (tp + fp)
  print '(A,F8.6)', 'recall=', tp / (tp + fn)
end program threshold_review
